Makefile

```diff
@@ -156,7 +156,10 @@ unit-cover: $(GOACC_BIN)
 
 unit-race:
 	@$(call print, "Running unit race tests.")
+	-mkdir app/build
```

Comment:
This is weird syntax. I think the dash isn't needed in front of the commands?
And I think this could be shortened to a single call: `mkdir -p app/build && touch app/build/index.html`
I don't think we need to clean up the build directory afterwards.

Reply:
updated :)